homeworks/rostyslav.havuka_RostyslavHavuka/13-ng-forms/forms/src/app/password-input-component/password-input-component.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, forwardRef, Input } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+
+@Component({
+  selector: 'app-password-input-component',
+  templateUrl: './password-input-component.component.html',
+  styleUrls: ['./password-input-component.component.css'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => PasswordInputComponentComponent),
+      multi: true
+    }
+  ]
+})
+export class PasswordInputComponentComponent  implements ControlValueAccessor{
+  value: string;
+  type: boolean;
```

Comment:
`type` means that you will have here a **type**, but not just `boolean`. So set here `string` or better `'text' | 'password'` type